common/test/acceptance/tests/studio/test_studio_split_test.py

```diff
@@ -482,6 +482,36 @@ def test_can_create_and_edit_group_configuration(self):
             groups=["First Group", "Group C", "Group D"]
         )
 
+    def test_focus_management_in_experiment_group_inputs(self):
+        """
+        Scenario: Ensure that selecting the focus inputs in the groups list sets the .is-focused class on the fieldset
```

Comment:
Nit: for some reason (legibility?) we keep docstrings to 80 characters, so I'd wrap this line.

I thought this was written down somewhere, but I can't find it here:

http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python_guidelines.html#docstrings